R/summaries.R

```diff
@@ -12,36 +12,54 @@
 #' @param method Specify the method to determine top taxa. Either sum, mean,
 #'   median or prevalence. Default is 'mean'.
 #'
-#' @param abund_values a \code{character} value to select an
+#' @param abund_values A \code{character} value to select an
 #'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
 #'
+#' @param ... Additional arguments not used.
+#'
 #' @details
-#' \code{getTopTaxa} extracts the most \code{top} abundant \dQuote{FeatureID}s
+#' The \code{getTopTaxa} extracts the most \code{top} abundant \dQuote{FeatureID}s
 #' in a \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
 #' object.
 #'
 #' @return
-#' For \code{getTopTaxa}: A vector of the most \code{top} abundant
+#' The \code{getTopTaxa} returns a vector of the most \code{top} abundant
 #' \dQuote{FeatureID}s
 #'
 #' @seealso
 #' \code{\link[=getPrevalence]{getPrevalentTaxa}}
 #'
-#' @name getTopTaxa
+#' @name summaries-basic
 #'
 #' @author
-#' Sudarshan A. Shetty
+#' Leo Lahti, Tuomas Borman and Sudarshan A. Shetty
 #'
 #' @examples
-#' data(GlobalPatterns)
+#' data("GlobalPatterns")
```

Comment:
I changed this to match the majority of uses.

`data` also accepts the name of the dataset, and does not require a `character` as input. More info: `?data`